src/apiserver/api_server_test.cc

```diff
@@ -475,6 +475,52 @@ TEST_F(APIServerTest, procedure) {
     ASSERT_TRUE(env->cluster_remote->ExecuteDDL(env->db, "drop table trans;", &status));
 }
 
+TEST_F(APIServerTest, getDBs) {
+    const auto env = APIServerTestEnv::Instance();
+    brpc::Controller show_cntl;  // default is GET
+    show_cntl.http_request().uri() = "http://127.0.0.1:8010/dbs";
+    env->http_channel.CallMethod(NULL, &show_cntl, NULL, NULL, NULL);
+    ASSERT_FALSE(show_cntl.Failed()) << show_cntl.ErrorText();
+    LOG(INFO) << "get dbs: " << show_cntl.response_attachment();
```

Comment:
How about add some dbs(some weird names) and check the results?

Reply:
Thanks, I will try it. But how to add more databases to APIServerTestEnv::Instance?

Reply:
`env->cluster_remote->CreateDB/DropDB` can do that.